opm/simulators/linalg/OwningTwoLevelPreconditioner.hpp

```diff
@@ -182,6 +182,7 @@ class OwningTwoLevelPreconditioner : public Dune::PreconditionerWithUpdate<Vecto
     CoarseSolverPolicy coarseSolverPolicy_;
     TwoLevelMethod twolevel_method_;
     boost::property_tree::ptree prm_;
+    Dune::Amg::SequentialInformation dummy_comm_;
```

Comment:
I am afraid that this change is not correct. If it compiles then that is just due to changed constructor never being called in paralllel. Please use the template parameter `Communication` as the type.

Reply:
The changed constructor should never be called in parallel. The constructor taking a communication object should be used in the parallel case. Using the wrong constructor is now a compilation error, but it is admittedly a confusing one.

Is it not possible to use Communication as type for the dummy either, as OwnerOverlapCopyCommunication cannot be default constructed (I may have misread of course).

Reply:
I guess you must have misread the source. There seems to be one in [owneroverlapcommunication.hh:582](https://gitlab.dune-project.org/core/dune-istl/blob/621e756d6a18a7591ecc0f1ae4bd84cd657ab7b1/dune/istl/owneroverlapcopy.hh#L582)

The other possibility is to always pass a communication object....

IMHO having obviously wrong (non-compilable) code seem like a no-go to me.

Reply:
> I guess you must have misread the source.

You are quite right, I did not notice the default argument. I will change this PR accordingly.

> The other possibility is to always pass a communication object....

That seems a bit akward in the serial case, what should that be?